src/main/java/com/stripe/model/reporting/ReportType.java

```diff
@@ -31,6 +32,10 @@
   @SerializedName("data_available_start")
   Long dataAvailableStart;
 
+  /** List of column names that are included by default when this Report Type gets run. */
+  @SerializedName("default_columns")
+  List<String> defaultColumns;
```

Comment:
Same for this; I didn't see it updated in 7.x

Reply:
yep this is new and was in the middle of API review so it's on my todo list